ansible/roles/csm.cn.rsyslog/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Hewlett Packard Enterprise Development LP
+---
+- name: Check and create rsyslog.d directory
+  file:
+    path: /etc/rsyslog.d
+    mode: '0755'
+    state: directory
+  when: cray_cfs_image|default(false)|bool or rsyslog_on_ncn|default(false)|bool
```

Comment:
This use of `cray_cfs_image`, while allowed, is discouraged. See here. file:///Users/jasonsollom/working/git/docs-csm/operations/configuration_management/Target_Ansible_Tasks_for_Image_Customization.md
Consider whether this can be incorporated into the hosts: line instead.

The OR conditional may make this tricky, so this is an uncertain suggestion at best. Same suggestion for line 13.